public/redesign/sass/main.scss

```diff
@@ -0,0 +1,5 @@
+@import "base";
```

Comment:
Not by convention. Use [single quotes](https://github.com/dev-labs-bg/devlabs-internal-code-conventions/blob/master/style.md#%D0%98%D0%B7%D0%BF%D0%BE%D0%BB%D0%B7%D0%B2%D0%B0%D0%B9%D1%82%D0%B5-%D0%B5%D0%B4%D0%B8%D0%BD%D0%B8%D1%87%D0%BD%D0%B8---%D0%BA%D0%B0%D0%B2%D0%B8%D1%87%D0%BA%D0%B8-%D0%BA%D0%BE%D0%B3%D0%B0%D1%82%D0%BE-%D1%81%D0%B5-%D0%BD%D0%B0%D0%BB%D0%B0%D0%B3%D0%B0-%D0%B4%D0%B0-%D0%BE%D0%B1%D0%B3%D1%80%D0%B0%D0%B4%D0%B8%D1%82%D0%B5-%D0%B0%D1%82%D1%80%D0%B8%D0%B1%D1%83%D1%82-%D1%81%D0%B5%D0%BB%D0%B5%D0%BA%D1%82%D0%BE%D1%80%D0%B8) 😸

Reply:
point taken :bowing_man: